user_events.py

```diff
@@ -535,6 +535,21 @@ def user_an_event_edit(id):
             update_user_event(id, post_by_id, {'endDate': ""})
         else:
             update_user_event(id, post_by_id, None)
+        if request.method == 'POST':
+            user_event = find_user_event(id)
+            if 'subEvents' in user_event and user_event['subEvents'] is not None:
+                for subevent in user_event['subEvents']:
+                    event = None
+                    if 'id' in subevent:
+                        event = find_one(current_app.config['EVENT_COLLECTION'],
+                                    condition={"platformEventId": subevent['id']})
+                    else:
+                        event = find_one(current_app.config['EVENT_COLLECTION'], condition={"_id": ObjectId(subevent['eventid'])})
+                    if event is not None and (not 'superEventID' in event):
```

Comment:
```suggestion
                    if event is not None and 'superEventID' not in event:
```
There was a PEP8 warning in the previous code. Suggesting code that follows PEP8. Thanks.